.github/workflows/weekly_batch.yml

```
@@ -1,13 +1,19 @@
 name: Weekly Book Production
 
 on:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
   schedule:
```

actionlint (1.7.7)

unexpected key "schedule" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)